validation/pipeline/tfp/Jenkinsfile.harvester.e2e

```diff
@@ -63,21 +64,141 @@ node("harvester-vpn-1") {
           
                 withEnv(paramsMap) {
 
+                    stage('Setup Harvester Environment') {
+                        sh returnStdout: true, script: 'wget -qO ./jq https://github.com/jqlang/jq/releases/latest/download/jq-linux-amd64'
+                        sh returnStdout:true, script: 'chmod a+x ./jq'
 
+                        sh returnStdout: true, script: 'wget -qO ./yq https://github.com/mikefarah/yq/releases/latest/download/yq_linux_amd64'
+                        sh returnStdout:true, script: 'chmod a+x ./yq'
+
+                        sh returnStdout: true, script:  'wget -qO ./kubectl "https://dl.k8s.io/release/\$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl"'
+                        sh returnStdout: true, script:  'chmod a+x ./kubectl'
```

Comment:
It would be good to create a function that installs all of these tools in the future. I'm sure this same pattern is used in other jobs.	

Sort of like this.

```
def setupTools() {
    // Download and configure required tools
    sh '''
        wget -qO ./jq https://github.com/jqlang/jq/releases/latest/download/jq-linux-amd64
        chmod a+x ./jq
        wget -qO ./yq https://github.com/mikefarah/yq/releases/latest/download/yq_linux_amd64
        chmod a+x ./yq
        wget -qO ./kubectl "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl"
        chmod a+x ./kubectl
    '''
}